ludwig/schema/trainer.py

```diff
@@ -207,6 +207,13 @@ def __post_init__(self):
         parameter_metadata=TRAINER_METADATA[MODEL_ECD]["train_steps"],
     )
 
+    eval_steps: float = schema_utils.NonNegativeInteger(
```

Comment:
Another option is to only support `eval_steps` for the LLMTrainer config. We're more concerned about slow synchronous evals for fine-tuning LLMs anyway. WDYT? CC: @geoffreyangus

Reply:
That's kinda what I was thinking, too